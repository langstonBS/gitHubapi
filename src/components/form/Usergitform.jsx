import React from 'react';
import PropTypes from 'prop-types';

const Usergetform = ({ userName, onChange, onSubmit }) => (
  <form onSubmit={onSubmit} >
    <input type="text" value={userName} onChange={onChange}></input>
    <button>Submit</button>
  </form>
);

Usergetform.propTypes = {
  userName: PropTypes.string.isRequired,
  onChange: PropTypes.func.isRequired,
  onSubmit: PropTypes.func.isRequired

};

export default Usergetform;
