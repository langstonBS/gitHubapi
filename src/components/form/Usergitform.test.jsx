import React from 'react';
import { shallow } from 'enzyme';
import Usergitform from './Usergitform';

describe('App component', () => {
  it('renders App', () => {
    const wrapper = shallow(<Usergitform userName="myName" onChange={() => { }} onSubmit={() => { }}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
