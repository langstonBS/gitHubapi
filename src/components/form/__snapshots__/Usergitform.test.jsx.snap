// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders App 1`] = `
<form
  onSubmit={[Function]}
>
  <input
    onChange={[Function]}
    type="text"
    value="myName"
  />
  <button>
    Submit
  </button>
</form>
`;
